src/main/java/com/google/sps/servlets/utils/CommonWordsRetriever.java

```diff
@@ -14,18 +14,33 @@
 
 package com.google.sps.servlets.utils;
 
+import java.io.File;
+import java.io.FileNotFoundException;
 import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.List;
+import java.util.Scanner;
 
-/** Class provides commonly used words that should be ignored within the word map */
+/** Class that provides commonly used words that should be ignored within the word map */
 public class CommonWordsRetriever {
-  // TODO: Parse text file for common words
-  private static final List<String> commonWordsList =
-      new ArrayList<>(
-          Arrays.asList(
-              "the", "it", "at", "and", "to", "we", "can", "are", "of", "is", "am", "i", "this",
-              "was"));
+  private static final List<String> commonWordsList = populateWordList();
+
+  /** Populate a list of strings from a text file containing the words to ignore in the word map */
+  private static List<String> populateWordList() {
+    List<String> commonWords = new ArrayList<String>();
+    try {
+      File file =
+          new File(
+              System.getProperty("user.dir")
+                  + "/src/main/java/com/google/sps/servlets/utils/resources/common_words.txt");
+      Scanner sc = new Scanner(file);
```

Comment:
You can this logic a lot by using [Files.readAllLines](https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#readAllLines(java.nio.file.Path,%20java.nio.charset.Charset)) (Which is also part of the java.io package) 

The main part of this logic would look like:

`return Files.readAllLlines(Paths.get(fileName), StandardCharsets.UTF_8);`